doap.rdf

```diff
@@ -0,0 +1,55 @@
+---
+---
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl"?>
+<rdf:RDF xml:lang="en"
+         xmlns="http://usefulinc.com/ns/doap#"
+         xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:asfext="http://projects.apache.org/ns/asfext#"
+         xmlns:foaf="http://xmlns.com/foaf/0.1/">
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+  <Project rdf:about="https://datafusion.apache.org">
+    <created>2024-04-17</created>
+    <license rdf:resource="https://spdx.org/licenses/Apache-2.0" />
+    <name>Apache DataFusion</name>
+    <homepage rdf:resource="https://datafusion.apache.org" />
+    <asfext:pmc rdf:resource="https://datafusion.apache.org" />
+    <shortdesc>Apache DataFusion is a very fast, extensible query engine for building high-quality data-centric systems.</shortdesc>
```

Comment:
I suggest we remove the `very` -- but it is a personal preference

I do think putting Rust in the short description is likely helpful to help people find the right place

```suggestion
    <shortdesc>Apache DataFusion is a fast, extensible query engine for building high-quality data-centric systems in Rust.</shortdesc>
```